atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -299,7 +308,7 @@ public void hostIsAutomaticallyRemovedOnStartup() {
         setCassandraServersTo(CASS_SERVER_1);
 
         createClientPool();
-        assertThat(poolServers).containsExactlyInAnyOrder(CASS_SERVER_1);
+        assertThatMap(poolServers).containsOnlyKeys(CASS_SERVER_1);
```

Comment:
Did not realise that there was a map specific method! you can just use assertThat though (since assertThat generates a MapAssert)

Reply:
oh yeah duh